lib/config/documentEventAliases.ts

```diff
@@ -22,12 +22,12 @@
 import { EventAliases } from '../types/config/EventAliases';
 import DocumentController from '../api/controllers/documentController';
 
-function filter (map: Map<string, string>, expectValue: string) {
+function filter (obj: Record<string, unknown>, expectValue: string): Array<string> {
   const result = [];
 
-  for (const [key, value] of map) {
-    if (value === expectValue) {
-      result.push(key);
+  for (const action in obj) {
```

Comment:
You should avoid `for in` loops https://bitsofco.de/for-in-vs-for-of/

Reply:
That's what I did, I didn't push yet.

Reply:
You can put the PR in draft mode then so other reviewers knows it still wip